subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/KotlinPluginSmokeTest.groovy

```diff
@@ -21,7 +21,7 @@ import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
 
 class KotlinPluginSmokeTest extends AbstractSmokeTest {
     private kotlinVersion = '1.1.1'
-    private androidPluginVersion = AndroidPluginsSmokeTest.ANDROID_PLUGIN_VERSION
+    private androidPluginVersion = '2.3.3'
```

Comment:
Updating this version number was forgotten before. Should this test maybe also use TESTED_ANDROID_PLUGIN_VERSIONS? Maybe TESTED_ANDROID_PLUGIN_VERSIONS.first() for now?

Reply:
Should we keep this a variable?